purchases/src/main/kotlin/com/revenuecat/purchases/google/errors.kt

```diff
@@ -12,6 +12,16 @@ internal fun @receiver:BillingClient.BillingResponseCode Int.getBillingResponseC
         ?: "$this"
 }
 
+@Suppress("MaximumLineLength", "MaxLineLength")
+internal fun @receiver:BillingClient.OnPurchasesUpdatedSubResponseCode Int.getOnPurchasesUpdatedSubResponseCodeName(): String {
+    val allPossibleOnPurchasesUpdatedSubResponseCodes =
+        BillingClient.OnPurchasesUpdatedSubResponseCode::class.java.declaredFields
+    return allPossibleOnPurchasesUpdatedSubResponseCodes
+        .firstOrNull { it.getInt(it) == this }
+        ?.name
+        ?: "$this"
+}
```

Comment:
I assume `getOnPurchasesUpdatedSubResponseCodeName` will barely get called but since it's using reflection, you could maybe create a static map that this function can access when called

Reply:
Well, it will get called on every purchase attempt... And yeah, might be better to create a map... I just followed what we do for the responseCode itself, but I think creating a map ourselves feels better yeah... Will do the change, and I might change that other mapping we do while I'm at it.